src/components/FormControl/FormControl.tsx

```diff
@@ -0,0 +1,72 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import InputLabel, { InputLabelProps } from "@components/InputLabel/InputLabel";
+import { FC, ReactElement, ReactNode } from "react";
+import css from "./FormControl.module.css";
+
+export enum HelperVariant {
+    Default = "Default",
+    Success = "Success",
+    Error = "Error",
+}
+
+type HelperTextProps = {
+    text: string;
+    disabled?: boolean;
+    variant?: HelperVariant;
+};
+const HelperText: FC<HelperTextProps> = ({ text, disabled, variant }: HelperTextProps) => (
+    <span
+        data-test-id="form-control-helper-text"
+        className={`${css.helper} ${disabled ? css.disabled : ""} ${
+            variant === HelperVariant.Error ? css.error : variant === HelperVariant.Success ? css.success : ""
+        }`}
+    >
+        {text}
+    </span>
+);
+
+export interface FormControlProps {
```

Comment:
The `HelperText` uses `type` for the prop-typing, whereas here it's an `interface`. I think we should avoid having a mixture of both. Personally I favour `type` over `interface` since we're only going for type-safety. Interfaces come from an OOP background, where classes implement `interfaces` so you can achieve loose coupling (is it Liskov substitution principle? I'm not sure after all these years without OOP :joy:). Interfaces can be extended and implemented, where as types are just that―types.

What do the two of you think? @dwirz @SamuelAlev

Reply:
I could agree with you as type composition looks nicer with `type` than `interface`.
And yes `interface` comes from an OOP background so it makes more sense.

Reply:
I suggest that after this one and https://github.com/Frontify/arcade/pull/7 is merged I open a new PR and align all the `Props` definitions